validator/project_validator.go

```diff
@@ -40,13 +40,16 @@ type ValidationErrorLevel int64
 const (
 	Error ValidationErrorLevel = iota
 	Warning
-	unauthorizedCharacters                  = "|"
 	EC2HostCreateTotalLimit                 = 1000
 	DockerHostCreateTotalLimit              = 200
 	HostCreateLimitPerTask                  = 3
 	maxTaskSyncCommandsForDependenciesCheck = 300 // this should take about one second
 )
 
+var (
+	unauthorizedCharacters = []string{"|", "&", ";", "$", "`", "'", "*", "?", "#", "%", "^", "@", "{", "}", "(", ")", "<", ">"}
```

Comment:
I think it could be worth commenting here or in the loop why we're using an ugly list instead of a cute regex, to avoid someone repeating the work you did (i.e. these characters are valid if they're unicoded)